### DISCUSSION TITLE
Help me configure my SPAN settings.

### DISCUSSION THREAD

**Discussion by SidssPT1** [Security-Onion-Solutions/securityonion#6305] [Unsupported Versions] [answered]
This may sound like a noob question but please help me out.

I have installed SO on Virtual Box. I have watched so many Security Onion installation and setup videos and in every such videos it is mentioned to configure 2 interface: 1 for management and 1 for SPAN.

I can setup SPAN port on my Switch and Firewall but what I don't understand is how to transfer SPAN port traffic to my Security Onion. I just simply need to forward SPAN port traffic to the NAT or bridged IP of my Security Onion or something else.

I have searched every where regarding this but haven't found any solution.

**Comment by Sahale** [comment]
What's host OS?

**Reply by SidssPT1** [reply]
No host OS. I have used the 7.8 GB ISO file to install security onion.

**Reply by Sahale** [reply]
You've installed SO in VirtualBox? VirtualBox installed on Windows or Linux VM?
Anyway, you should set your SO monitor interface to "bridged" connection to your physical span interface and enable promiscuous mode. It'll work in Linux, but I don't know about Windows.

**Reply by SidssPT1** [reply]
VM is installed on Windows Server 2019. Thanks for the help.

**Comment by tomjones1977** [comment, marked as answer]
If you have a SPAN port on your switch, you will need to physically connect it to a network interface on your PC.  You then need to bridge that NIC to your SecOnion VM and enable promiscuous mode.  Note that this still may not work depending on the setup.  Another option would be to pass through a USB NIC direct to the VM and allow SO to handle the configuration.  This skips two or three layers of network abstraction.

**Reply by SidssPT1** [reply]
Thanks for the detailed answer. But what if I have multiple switches in my network.

**Reply by tomjones1977** [reply]
If you have multiple switches you need to monitor you have a couple of options (depending on the capability of the switches).  One is to put a forward node on each switch and run a SPAN port on each switch individually.  However this takes a lot of hardware.

Another is to configure RSPAN (if available) which will send the SPAN data to another device.  Cisco example: https://community.cisco.com/t5/networking-documents/understanding-span-rspan-and-erspan/ta-p/3144951

If you can live without full packet capture you could configure Netflow to get source/dest IP and port data.  There is a good video from the SecOnion team that has been published recently on this: https://www.youtube.com/watch?v=ew5gtVjAs7g
